client/src/components/AddVideo.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+
+const AddVideo = ({ data, setData }) => {
+    const [displayAddVideo, setDisplayAddVideo] = useState(false);
+    const [title, setTitle] = useState("");
+    const [url, setUrl] = useState("");
+
+    const handleAdd = (e) => {
+        const videoId = Math.floor(Math.random() * 100000);
```

Comment:
Hi Askin, I suggest that check this link for a unique ID for your videos.
https://www.npmjs.com/package/uuid

Reply:
Hi Yunus, many thanks for the advice. I have installed uuid library to use unique id for each new videos.